CRITICAL - 2025-05-07 17:25:44 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:25:44 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:26:00 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:26:04 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:26:08 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:26:10 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:26:10 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(12): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
CRITICAL - 2025-05-07 17:27:08 --> ErrorException: Required @OA\PathItem() not found
[Method: GET, Route: swagger]
in VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php on line 23.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\PathItem() not found', '/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php', 23)
 2 VENDORPATH/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\PathItem() not found', 512)
 3 VENDORPATH/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\PathItem() not found', [])
 4 VENDORPATH/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\PathItem() not found')
 5 VENDORPATH/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
 6 VENDORPATH/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
 7 VENDORPATH/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate([...], Object(OpenApi\Analysis), true)
 8 APPPATH/Controllers/Swagger.php(34): OpenApi\Generator::scan([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Swagger->index()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH/rewrite.php(44): require_once('/Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php')
ERROR - 2025-05-07 17:28:11 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:14 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:14 --> Stack trace: #0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\Pa...', '/Users/joaodell...', 23)
#1 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\Pa...', 512)
#2 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\Pa...', Array)
#3 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\Pa...')
#4 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
#5 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
#6 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate(Array, Object(OpenApi\Analysis), true)
#7 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/app/Controllers/Swagger.php(49): OpenApi\Generator::scan(Array)
#8 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Swagger->index()
#9 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
#10 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/joaodell...')
#15 {main}
ERROR - 2025-05-07 17:29:15 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:15 --> Stack trace: #0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\Pa...', '/Users/joaodell...', 23)
#1 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\Pa...', 512)
#2 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\Pa...', Array)
#3 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\Pa...')
#4 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
#5 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
#6 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate(Array, Object(OpenApi\Analysis), true)
#7 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/app/Controllers/Swagger.php(49): OpenApi\Generator::scan(Array)
#8 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Swagger->index()
#9 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
#10 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/joaodell...')
#15 {main}
ERROR - 2025-05-07 17:29:16 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:16 --> Stack trace: #0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\Pa...', '/Users/joaodell...', 23)
#1 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\Pa...', 512)
#2 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\Pa...', Array)
#3 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\Pa...')
#4 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
#5 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
#6 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate(Array, Object(OpenApi\Analysis), true)
#7 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/app/Controllers/Swagger.php(49): OpenApi\Generator::scan(Array)
#8 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Swagger->index()
#9 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
#10 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/joaodell...')
#15 {main}
ERROR - 2025-05-07 17:29:18 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:18 --> Stack trace: #0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\Pa...', '/Users/joaodell...', 23)
#1 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\Pa...', 512)
#2 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\Pa...', Array)
#3 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\Pa...')
#4 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
#5 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
#6 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate(Array, Object(OpenApi\Analysis), true)
#7 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/app/Controllers/Swagger.php(49): OpenApi\Generator::scan(Array)
#8 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Swagger->index()
#9 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
#10 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/joaodell...')
#15 {main}
ERROR - 2025-05-07 17:29:23 --> Swagger generation error: Required @OA\PathItem() not found
ERROR - 2025-05-07 17:29:23 --> Stack trace: #0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(512, 'Required @OA\\Pa...', '/Users/joaodell...', 23)
#1 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Loggers/DefaultLogger.php(23): trigger_error('Required @OA\\Pa...', 512)
#2 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/psr/log/src/LoggerTrait.php(61): OpenApi\Loggers\DefaultLogger->log('warning', 'Required @OA\\Pa...', Array)
#3 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Annotations/OpenApi.php(157): Psr\Log\AbstractLogger->warning('Required @OA\\Pa...')
#4 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Analysis.php(423): OpenApi\Annotations\OpenApi->validate()
#5 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(377): OpenApi\Analysis->validate()
#6 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/zircote/swagger-php/src/Generator.php(320): OpenApi\Generator->generate(Array, Object(OpenApi\Analysis), true)
#7 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/app/Controllers/Swagger.php(49): OpenApi\Generator::scan(Array)
#8 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Swagger->index()
#9 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Swagger))
#10 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/public/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 /Users/joaodellarmelina/repo/new Api/relatorio-jade-api/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('/Users/joaodell...')
#15 {main}
